packages/dd-trace/test/appsec/rule_manager.spec.js

```diff
@@ -299,7 +303,13 @@ describe('AppSec Rule Manager', () => {
               'category': 'attack_attempt',
               'confidence': '1'
             },
-            'conditions': []
+            'conditions': [{
```

Comment:
Added values in `'conditions'` because the waf was throwing an exception, and `rulesData` were not applying. And we were ignoring that, now we are checking that the `apply_state` is `ACKNOWLEDGED`